src/odm_sharing/private/cons.py

```diff
@@ -223,6 +224,12 @@ def _detect_sqlite(path: str) -> bool:
         return False
 
 
+def _detect_sqlalchemy(path: str) -> bool:
+    # https://docs.sqlalchemy.org/en/20/core/engines.html#database-urls
+    # regex checks if path starts with '<dialect>[+<driver>]://'
+    return bool(re.match(r'\w+(\+\w+)?:\/\/', path))
```

Comment:
Can we use this method to detect sqlalchemy URL https://docs.sqlalchemy.org/en/21/core/engines.html#sqlalchemy.engine.make_url? It looks like it throws an error if its not a valid URL.

Reply:
done